Lab-meetings.md

```diff
@@ -0,0 +1,347 @@
+# Lab Meetings
+
+As the lab grows, the importance of sharing technical skills, domain expertise and tips and tricks for working effectively grows with it.
+Lab meetings are a space for all members of the group to come together and learn from each other.
+
```

Comment:
I think a section on the aims or outcomes of a lab meeting would be useful here

Reply:
Now that you've heard me present in the meeting, can you give adding those sentences a bash @ajrcampbell?